accessibility-checker-extension/src/ts/devtools/components/reportSection.tsx

```diff
@@ -287,8 +287,8 @@ export class ReportSection extends React.Component<ReportSectionProps, ReportSec
                         inline={true}
                         onClick={() => {
                             let appController = getDevtoolsAppController();
-                            appController.setSecondaryView("summary");
-                            appController.openSecondary("totalIssuesCount");
```

Comment:
Something doesn't look right here

Reply:
Fixed somehow the appController.setSecondaryView before ("summary") got deleted. It is back now.

Reply:
Also fixed the Mask group issues.

Reply:
All ready now except for the bee icon for the popup. This is in the manifest.json file and I don't know how to do a browser detection in the manifest file.